packages/mc-scripts/src/bin/cli.js

```diff
@@ -94,6 +94,10 @@ const applicationDirectory = fs.realpathSync(process.cwd());
         proxyCommand(command);
         break;
       }
+      case 'login': {
+        proxyCommand(command);
+        break;
+      }
```

Comment:
Please also set the `NODE_ENV` variable as in the other commands. I think `production` should be fine.